webpack.config.js

```diff
@@ -174,6 +174,28 @@ var info = {
 
         "machines/index.html",
         "machines/manifest.json",
+        "machines/vnc.html",
+        "machines/include/base64.js",
+        "machines/include/des.js",
+        "machines/include/display.js",
+        "machines/include/inflator.js",
+        "machines/include/input.js",
+        "machines/include/keyboard.js",
+        "machines/include/keysymdef.js",
+        "machines/include/keysym.js",
+        "machines/include/logo.js",
+        "machines/include/playback.js",
+        "machines/include/rfb.js",
+        "machines/include/ui.js",
+        "machines/include/util.js",
+        "machines/include/websock.js",
+        "machines/include/webutil.js",
+
+        "machines/include/Orbitron700.ttf",
+        "machines/include/Orbitron700.woff",
+        "machines/include/base.css",
+        "machines/include/black.css",
+        "machines/include/blue.css",
```

Comment:
This repeats that long and changing list another time. Like above, would be better to let a function do it by iterating over include/.